docs/mm_bot/architecture.md

```diff
@@ -36,30 +40,37 @@ The bot architecture is as follows:
 
 ![architecture.png](images/architecture.png)
 
-The bot is composed of the following components:
-- **Main Process**: The main process of the bot. It has the following subcomponents:
-    - `Main Order Indexer`: The `Main Order Indexer` is responsible for indexing the orders from 
-    the pending blocks.
-  - `Order Processor`: Responsible for processing the orders.
-  - `Failed Orders Processor`: Responsible for retrying the failed orders.
-  It runs every 5 minutes.
-  - `Accepted Blocks Processor`: Responsible for indexing the orders that belong to accepted blocks. It runs every 5 minutes.
-- **Database**: The database is used to store the following data:
-  - Orders
-  - Errors
-  - Block numbers
-
-An important aspect of the bot is that it must be able to handle multiple orders concurrently.
+An `Orders Processor`  is needed to process the orders for each L2.
```

Comment:
I think this explains better the 1-1 relationship between Orders Processor and L2.
```suggestion
One `Orders Processor` is needed to process the orders for each L2.
```